deploy.sh

```diff
@@ -115,9 +115,11 @@ svn cp "trunk" "tags/$VERSION"
 
 # Fix screenshots getting force downloaded when clicking them
 # https://developer.wordpress.org/plugins/wordpress-org/plugin-assets/
-svn propset svn:mime-type image/png assets/*.png || true
-svn propset svn:mime-type image/jpeg assets/*.jpg || true
-svn propset svn:mime-type image/gif assets/*.gif || true
+if test -d $ASSETS_DIR && test -n "$(find $ASSETS_DIR -maxdepth 1 -print -quit)"; then
+	svn propset svn:mime-type image/png assets/*.png || true
+	svn propset svn:mime-type image/jpeg assets/*.jpg || true
+	svn propset svn:mime-type image/gif assets/*.gif || true
+fi
```

Comment:
Hi @Lewiscowles1986, Thanks a lot for the PR. 

I think checking for files by extensions is a better approach here, because with this check if the repo has only `.png` images then it will still show the warning for command related to `.jpg` and `.gif` as the repo is not containing those files. 
So, I think we can do something like below to check for files with specific extensions and run the command only if file is present. what do you think?
```
if test -d $ASSETS_DIR && test -n "$(find $ASSETS_DIR -maxdepth 1 -name *.png -print -quit)"; then
	svn propset svn:mime-type image/png assets/*.png || true
fi
if test -d $ASSETS_DIR && test -n "$(find $ASSETS_DIR -maxdepth 1 -name *.jpg -print -quit)"; then
	svn propset svn:mime-type image/jpeg assets/*.jpg || true
fi
if test -d $ASSETS_DIR && test -n "$(find $ASSETS_DIR -maxdepth 1 -name *.gif -print -quit)"; then
	svn propset svn:mime-type image/gif assets/*.gif || true
fi
```
Thanks.

Reply:
Can we also address the change introduced in #78 here as well?

Reply:
Got a notification that shellcheck failed, so addressed it's feedback to quote inputs. Also quoted the mimes and globs.

Reply:
Checked manually. It now passes shellcheck.